src/Features/Core/Portable/Completion/Providers/AbstractInternalsVisibleToCompletionProvider.cs

```diff
@@ -243,6 +248,49 @@ public override async Task<CompletionChange> GetChangeAsync(Document document, C
             return CompletionChange.Create(textChange);
         }
 
+        private static async Task<TextSpan> GetTextChangeSpanAsync(Document document, TextSpan startSpan, CancellationToken cancellationToken)
+        {
+            var result = startSpan;
+            var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var token = root.FindToken(result.Start);
+            if (syntaxFacts.IsStringLiteral(token) || syntaxFacts.IsVerbatimStringLiteral(token))
+            {
+                result = token.Span;
+                var contentStartsAt = token.Text.IndexOf('"');
+                contentStartsAt = contentStartsAt >= 0
+                    ? contentStartsAt + 1
+                    : contentStartsAt;
+                var contentEndsAt = token.Span.Length == 0
+                    ? token.Span.End
+                    : token.Text[token.Span.Length - 1] == '"'
+                        ? token.Span.End - 1
+                        : await GetEndOfOpenEndedStringLiteral(startSpan.End).ConfigureAwait(false); // The string is open ended. We keep the right end as is.
+                result = TextSpan.FromBounds(
+                    start: token.Span.Start + contentStartsAt,
+                    end: contentEndsAt);
+            }
+
+            return result;
+
+            async Task<int> GetEndOfOpenEndedStringLiteral(int startPosition)
+            {
+                var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+                var end = startPosition;
+                while (end < text.Length)
+                {
+                    var ch = text[end];
+                    if (char.IsWhiteSpace(ch)) // this includes new line characters
+                    {
+                        break;
+                    }
+                    end++;
+                }
+
+                return end;
```

Comment:
@CyrusNajmabadi you proposed to do:

>In fact, to make your life easier. Just call: CommonCompletionUtilities.GetWordSpan. However, for the isWordCharacter helper pass: ch => SyntaxFacts.IsIdentifierPartCharacter(ch) || ch == '.'

`CommonCompletionUtilities.GetWordSpan` moves the end further only if the char before the cursor is a word character. Therefore I decided to make a copy of the relevant code. I do also think that we should not rely on `IsIdentifierPartCharacter` here, because we expect an assembly name with an optional `, PublicKey=hash` appendix. So a proper character list would be 
* all chars allowed in an assembly name
* comma, space, and equals sign

Reply:
> moves the end further only if the char before the cursor is a word character. Therefore I decided to make a copy of the relevant code.

I'd much rather us tweak GetWordSpan so it can handle that case.  something like `bool alwaysExtendEndSpan = false`.

Reply:
> So a proper character list would be...

Sure, that seems reasonable to me.  :)

Reply:
Done in 30340f4.

Reply:
I only use it to find the end. To find the start I do start at the begin of the token.Span to avoid problems with escape sequences.